docs/en/reference/pipeline-integrations/toolkit-engines-supported-os-versions.md

```diff
@@ -13,4 +13,4 @@ Here is a list of Toolkit engine supported operating systems.
 
 | Windows | macOS | CentOS |
 |:-------:|:-----:|:------:|
-|   10    |10.14+ <br>**Note:** M1 is not supported yet, [learn more here](https://community.shotgridsoftware.com/t/13606).    |  7.4   |
\ No newline at end of file
+|   10    |10.14+ <br>**Note:** M1 native is not supported, M1 with Rosetta is supported.    |  7.4   |
```

Comment:
One question... why you removed link to community post about how to enable Rosetta in Photoshop?